airbyte-integrations/connectors/source-github/integration_tests/configured_catalog.json

```diff
@@ -381,6 +381,16 @@
       "sync_mode": "incremental",
       "destination_sync_mode": "overwrite"
     },
+    {
+      "stream": {
+        "name": "workflow_jobs",
+        "json_schema": {},
+        "supported_sync_modes": ["full_refresh", "incremental"],
+        "source_defined_primary_key": [["id"]]
```

Comment:
I guess that is possible to run incremental based on id as these records should not change??

Reply:
The workflow jobs is a nested stream since you need to have the workflow_run to query it. But I followed the recommendations I found here: https://discuss.airbyte.io/t/how-does-sync-mode-work-with-sub-stream-nested-stream/587/3